apps/builder/app/builder/features/sidebar-left/panels/pages/page-settings.tsx

```diff
@@ -306,7 +309,34 @@ const PathField = ({
             </Tooltip>
           </Flex>
         ) : (
-          "Path"
+          <Flex align="center" gap={1}>
+            Path
+            <Tooltip
+              content={
+                <>
+                  <Text>
+                    The path can include dynamic parameters like :name in Pro.
```

Comment:
Just gave it another shot:

```suggestion

Path is a subset of the URL that looks like this: "/blog". To make the path dynamic and use it with CMS, you can use parameters and other features. CMS features are part of the Pro plan.
```